src/ceph-volume/ceph_volume/tests/devices/lvm/test_batch.py

```diff
@@ -1,11 +1,61 @@
+import mock
```

Comment:
there is no need for mock, we use py.test built in monkeypatching tool: `monkeypatch`. There are a lot of different places we use it in ceph-volume tests, you can look at those or follow the docs here https://docs.pytest.org/en/latest/monkeypatch.html